### ISSUE TITLE
Customize boolean emoji with locales

### ISSUE THREAD

**Issue by neumayr** [zooppa/administrate-field-boolean_emoji#1]
Ciao @michelegera 

I would like to switch to this `administrate-field` plugin if 
customization of the boolean output is possible.
What do you think about moving the default to a locale, like:

```yml
administrate:
  fields:
    boolean_emoji:
      'false': "🚫"
      'true': "✅"
```

This would also allow overriding the default values with normal Strings.
I would volunteer to submit a PR if you agree.

**Comment by michelegera** [comment]
@neumayr great idea, please go ahead with the PR!

**Comment by michelegera** [comment]
Closed via c88170573aedc734963343e2b21c64c6e78de40c.
